CHANGELOG.md

```diff
@@ -1022,4 +1022,7 @@ FEATURES:
   https://nfs.us-mci.ionos.com (Lenexa, USA)
 
 ### Fixes:
-- Changed Dead link in MongoDB cluster docs
\ No newline at end of file
+- Changed Dead link in MongoDB cluster docs
```

Comment:
@cciopa-ionos I think this was added in another PR. All changes that are related to documentation, such as this one, should be under `### Docs` chapter. You can leave this as it is but keep this in mind for the future. Under `### Fixes` we usually write what we actually fixed in the code.